pkg/preflight/assets/preflights.yaml

```diff
@@ -19,3 +20,12 @@ spec:
               message: "Not all nodes are online."
           - pass:
               message: "All nodes are online."
+    - clusterPodStatuses:
+        checkName: Pods status check
+        exclude: '{{kurl or (not .IsPrimary) (not .IsUpgrade) }}'
```

Comment:
@laverya @emosbaugh WDYT about this one?

Reply:
Does this apply to all pods in the cluster? If so I am opposed as non-infrastructure pods should no impact the install.

Reply:
I added the namespaces now,
If the namespace does not exist that does not fail